apiserver/openapi/oim_itsm.yaml

```diff
@@ -19,6 +19,12 @@ paths:
             application/json:
               schema:
                 $ref: "#components/schemas/update_change_response"
+        500:
+          description: "Server error"
+          content:
+            application/json:
+              schema:
+                $ref: "#components/schemas/update_change_response"
```

Comment:
This seems wrong, the content of the error contains the same variables as a normal result? Surely not... I'd recommend removing the content key here.

Reply:
Is changed